source/authentication/dex.rst

```diff
@@ -59,11 +59,10 @@ OnDemand Dex is configured by modifying the Open OnDemand Portal :ref:`ood-porta
 
 The default location for Dex configurations is :file:`/etc/ood/dex/config.yaml`.
 
-When changes are needed for OnDemand Dex run the following command:
+When changes are needed for OnDemand Dex :ref:`restart-apache` and then restart dex with:
 
    .. code-block:: sh
 
-      sudo /opt/ood/ood-portal-generator/sbin/update_ood_portal
```

Comment:
Bouncing apache runs `update_ood_portal` on all operating systems. I'm of the mind we should just instruct folks to systemd unit restarts instead of having to remember `update_ood_portal` on top of it.

Reply:
For manual installs, this is true, but for Puppet I remove the unit override to restart update_ood_portal since Puppet handles that operation as Puppet knows when it is necessary to run that command.  Not sure if that means this should stay or just assume people are using the behavior we deploy.

Reply:
In both cases, it's essentially handled by some other thing is my point.  At this point it's internal to either the default or the automation.

Reply:
That's fine.  As long as the script is mentioned somewhere else, seems fine to remove it from places where we also tell people to restart Apache.